server/routes/additionalDays/numberOfAdditionalDays/numberOfAdditionalDaysPage.ts

```diff
@@ -106,7 +106,7 @@ export default class NumberOfAdditionalDaysPage {
       return adjudicationUrls.isPunishmentSuspended.urls.edit(adjudicationNumber, req.params.redisId)
     }
     if (this.pageOptions.isManualEdit()) {
-      return adjudicationUrls.whichPunishmentIsItConsecutiveToManual.urls.edit(adjudicationNumber, req.params.redisId)
+      return adjudicationUrls.whichPunishmentIsItConsecutiveToManual.urls.start(adjudicationNumber)
```

Comment:
Assumed we'd want to switch to using `whichPunishmentIsItConsecutiveToManual.start` as `edit` goes into [this block](https://github.com/ministryofjustice/hmpps-manage-adjudications/blob/79ba2403cbcaaeb7a953630007501809fa950fff/server/routes/additionalDays/whichPunishmentConsecutiveTo/whichPunishmentConsecutiveToPage.ts#L76-L82)